README.md

```diff
@@ -1,31 +1,46 @@
-# START TO CREATING A COMBINED README
-
 # Apollo Template for ApostropheCMS + Astro integration
 
 This is intended as both a template and starting point for a project with an [ApostropheCMS](https://docs.apostrophecms.org/) backend and frontend powered by [Astro](https://astro.build/).
 
 The ApostropheCMS codebase is located in the `backend` folder of the repository, while the Astro codebase is in the `frontend` folder.
 
 ## Introduction
-Overall, this project utilizes ApostropheCMS as a headless backend with Astro as a frontend. What sets this apart from the typical headless use of ApostropheCMS is the addition of a package, [apostrophe-astro](https://github.com/apostrophecms/apostrophe-astro) in the Astro frontend project. This allows for full use of the ApostropheCMS Admin UI, including in-context editing. At the same time, this package also largely automates and simplifies fetching content from the ApostropheCMS backend without having to make REST API calls.
+Overall, this project utilizes ApostropheCMS as a headless backend with Astro as a frontend. What sets this apart from the typical headless use of ApostropheCMS is the addition of a package, [apostrophe-astro](https://github.com/apostrophecms/apostrophe-astro) in the Astro frontend project. This allows for full use of the ApostropheCMS Admin UI, including in-context editing. At the same time, this package also largely automates and simplifies fetching content from the ApostropheCMS backend without write REST API calls into your code.
 
 ## Using this project
-As outlined briefly above, both this repo and the `apollo-frontend` projects need to be provided with the `APOS_EXTERNAL_FRONT_KEY` environment variable set to the same value. Then, you can start the project as you would any ApostropheCMS project. For example, in a local development environment you can use `npm run dev`. For more details on working with the code in an ApostropheCMS project you can examine our [documentation](https://docs.apostrophecms.org/).
 
-### Similarities to a stand-alone project
-If you have worked with an ApostropheCMS project previously, this repo should look as expected. There are a number of custom modules, providing new pieces, pages, and widgets, located in the `modules` folder. There are also a number of modules that are improving the core Apostrophe modules located in the `modules/@apostrophecms` folder. For a full understanding of Apostrophe you should consult the [documentation](https://docs.apostrophecms.org/), but we will touch on a few highlights later in this document.
+### Prerequisites
+- Node.js v20 or later
+- MongoDB v4.4 or later running on a local server or access to Atlas. See the [ApostropheCMS documentation](https://docs.apostrophecms.org/guide/development-setup.html) for set-up details.
```

Comment:
6.0 is currently the oldest supported version upstream, so specify that, although it's true Apostrophe won't fall over on 4.4 (probably)

Reply:
Huh, where did I come up with 4.4. Oddly specific! Changed.